client_test/volume_test.py

```diff
@@ -0,0 +1,82 @@
+# Copyright Modal Labs 2023
+
+import pytest
+
+import modal
+from modal.exception import InvalidError
+from modal.volume import VolumeHandle
+
+from .supports.skip import skip_windows
+
+
+def dummy():
+    pass
+
+
+def test_volume_mount(client, servicer):
+    stub = modal.Stub()
+
+    dummy_modal = stub.function(
+        volumes={"/root/foo": modal.Volume()},
+    )(dummy)
+
+    with stub.run(client=client):
+        dummy_modal.call()
+
+
+@skip_windows("TODO: implement client-side path check on Windows.")
+def test_volume_bad_paths(client, servicer):
+    stub = modal.Stub()
+
+    dummy_modal = stub.function(volumes={"/root/../../foo": modal.Volume()})(dummy)
+    with pytest.raises(InvalidError):
+        with stub.run(client=client):
+            dummy_modal.call()
```

Comment:
I think you should be able to skip the function calls and just do `pass` within the run blocks, as all the loading logic should reside within the setup of app/context manager anyways